app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -121,15 +113,6 @@ export default function EditProfileForm() {
       ) : null}
       {user ? (
         <>
-          <form onSubmit={onSubmit}>
-            <ProfileTextInput
-              label="Name"
-              name="name"
-              defaultValue={user.name}
-              inputRef={register}
-              className={classes.field}
-            />
```

Comment:
Maybe we should keep this, or allow it to be changed on the other page. There is/will be a separate field for  "legal name" used for verification purposes.

Reply:
Should we call this "display name" or something instead?